package.json

```diff
@@ -35,6 +35,13 @@
   },
   "scripts": {
     "build": "webpack",
-    "build:watch": "webpack --watch --progress"
+    "build:watch": "webpack --watch --progress",
+    "test:frontend": "node_modules/.bin/babel-tape-runner 'web-external/tests/**/*.js*' | node_modules/.bin/tap-spec"
```

Comment:
You can just say `babel-tape-runner` and `tap-spec` here; NPM knows to look in `node_modules` for these executables.